utils/db/trigger.ts

```diff
@@ -12,12 +12,31 @@ export const getUserInfoFromDb = async (email: string) => {
 		throw new Error("Error in running the query on the database", err);
 	});
 	if (result.rows.length === 0) {
-		throw new Error('No topic found');
+		throw new Error('No user found');
 	}
     const userId: string = result.rows[0].id;
     const topicName: string = result.rows[0].topic_name;
 	return {
         userId,
         topicName,
     };
+}
+
+export const getRepoConfig = async (provider: string, repoName: string, repoOwner: string, userId: string) => {
+    console.info(`[getRepoConfig] Getting repo config for user: ${userId} and repo: ${repoName}`);
+    const query = `
+    SELECT config
```

Comment:
The query seems to incorrect.

Maybe something like this would be better:
```
json_build_object(
			'auto_assign', rc.auto_assign,
			'comment', rc.comment_setting
		) AS config
```

Reply:
done